### PR TITLE
allow offscreen rendering of trend chart

### PR DESCRIPTION
**Pull request by stevecrozz**

### CODE REVIEW

File: app/assets/javascripts/trend.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      ]
+    }
+
+    var origHeight = canvas.attr('height');
+    var origWidth = canvas.attr('width');
+
+    chart = new Chart(ctx);
+
+    // hack to allow offscreen rendering using preset height/width
+    if (typeof origHeight !== 'undefined' && typeof origWidth !== 'undefined') {
```
Comment (ontarionick) [inline app/assets/javascripts/trend.js:24]: lol ¯_(ツ)_/¯

**Comment by ontarionick** [comment]
:+1:

**Comment by stevecrozz** [comment]
Excellent. Just click the merge button whenever you're ready and this change should appear in the other PR.

**Comment by ontarionick** [comment]
Sorry, was just testing it out locally

**Comment by stevecrozz** [comment]
Sure, no prob. Didn't find any issues?

**Comment by ontarionick** [comment]
Looked great! Still not sure why the original version worked but my first attempt didn't, but it looks like this fixes it

**Pull request merged**